### PR TITLE
fix browserify build

### LINKED ISSUE #44: Remove require of encoding-indexes.js
Right now `encoding-indexes.js` is always required in a node like environment (such as browserify).

This was done in a85ce6ff385b71a9cafdad46cb6965ff3bd2de36

It would be better if `encoding-indexes.js` was never implicitly included.

### PR DESCRIPTION
**Pull request by calvinmetcalf**
this fixes #44 by doing 2 things

1. making it so that encoding-indexes doesn't actually export anything (all it was exporting was global anyway) and conversely encodings doesn't do anything beside just import.
2. adds a key to the package.json which prevents loading of encoding-indexes in browserify/webpack

### CODE REVIEW

File: lib/encoding.js [hunk 1]
```diff
@@ -11,8 +11,7 @@
   // If we're in node require encoding-indexes and attach it to the global.
   if (typeof module !== "undefined" && module.exports &&
     !global["encoding-indexes"]) {
-    global["encoding-indexes"] =
-      require("./encoding-indexes.js")["encoding-indexes"];
+      require("./encoding-indexes.js");
```
Comment (inexorabletash) [inline lib/encoding.js]: This directly conflicts with the change made for b4e5bc26e26e51f56e3daa9f13138c79f49d3c34 for #67

File: lib/encoding-indexes.js [hunk 2]
```diff
@@ -1,10 +1,6 @@
 (function(global) {
   'use strict';
 
-  if (typeof module !== "undefined" && module.exports) {
-    module.exports = global;
-  }
-
   global["encoding-indexes"] =
```
Comment (calvinmetcalf) [inline lib/encoding-indexes.js]: the idea was that any export was unnecessary due to this

**Comment by inexorabletash** [comment]
@therufa - can you comment on this?

**Comment by calvinmetcalf** [comment]
I just published my fork as text-encoding-polyfill

**Pull request closed without merging**